www/front_src/src/components/Hook/index.tsx

```diff
@@ -28,8 +28,8 @@ const LoadableHooks = ({
   return (
     <>
       {Object.entries(hooks)
-        .filter(([hook]) => hook === path)
-        .map(([, parameters]) => {
+        .filter(([hook]) => hook.includes(path))
+        .map(([_, parameters]) => {
```

Comment:
what is the "_" ?

Reply:
Because we want to ignore the first entries of the array. 
It is the same syntax in Golang